pennylane/math/single_dispatch.py

```diff
@@ -103,6 +108,8 @@ def _cond(pred, true_fn, false_fn, args):
 # qml.numpy rather than autograd.numpy, to take into account our autograd modification.
 ar.autoray._MODULE_ALIASES["autograd"] = "pennylane.numpy"
 
+ar.register_function("autograd", "ndim", lambda x: _i("qml").numpy.ndim(x))

```

Comment:
I think that should work with "autograd" instead of "qml".

Reply:
using autograd now.